src/seedu/addressbook/data/person/Email.java

```diff
@@ -6,7 +6,7 @@
  * Represents a Person's email in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidEmail(String)}
  */
-public class Email {
+public class Email implements Printable {
 
     public static final String EXAMPLE = "valid@e.mail";
     public static final String MESSAGE_EMAIL_CONSTRAINTS =
```

Comment:
Hi Justin, erm is that a missing chunk of code i'm looking at o.O? (after the "=" sign)